README.md

```diff
@@ -1,45 +1,33 @@
-
-## Usage
-
-Create a new repository from this template and run `./rename.js` specifying the new name of the project and the description to use e.g.
-```
-./rename.js ffc-demo-web "Web frontend for demo workstream"
-```
-
-The script will update the following:
-
-* `package.json`: update `name`, `description`, `homepage`
-* `docker-compose.yaml`: update the service name, `image` and `container_name`
-* `docker-compose.test.yaml`: update the service name, `image` and `container_name`
-* `docker-compose.override.yaml`: update the service name, `image` and `container_name`
-* Rename `helm/ffc-template-node`
-* `helm/ffc-template-node/Chart.yaml`: update `description` and `name`
-* `helm/ffc-template-node/values.yaml`: update  `name`, `namespace`, `workstream`, `image`, `containerConfigMap.name`
-* `helm/ffc-template-node/templates/_container.yaml`: update the template name
-* `helm/ffc-template-node/templates/cluster-ip-service.yaml`: update the template name and list parameter of include
-* `helm/ffc-template-node/templates/config-map.yaml`: update the template name and list parameter of include
-* `helm/ffc-template-node/templates/deployment.yaml`: update the template name, list parameter of deployment and container includes
-
-### Notes on automated rename
-
-* The Helm chart deployment values in `helm/ffc-template-node/values.yaml` may need updating depending on the resource needs of your microservice
-* The rename is a one-way operation i.e. currently it doesn't allow the name being changed from to be specified
-* There is some validation on the input to try and ensure the rename is successful, however, it is unlikely to stand up to malicious entry
-* Once the rename has been performed the script can be removed from the repo
-* Should the rename go awry the changes can be reverted via `git clean -df && git checkout -- .`
+This service is part of the [Single Front Door (SFD) service](https://github.com/DEFRA/fcp-fd-core).
 
 ## Prerequisites
-
 - Docker
 - Docker Compose
 
 Optional:
 - Kubernetes
 - Helm
 
+## Setup
+
+### Configuration
+
+These configuration values should be set in the docker-compose.yaml file or Helm values.yaml file if deploying to Kubernetes.
+
+| Name                      | Default Value                                          | Required                  | Description                                                                 |
+|---------------------------|--------------------------------------------------------|---------------------------|-----------------------------------------------------------------------------|
+| USE_AZURITE               | false                                                  | No                        | Flag to use Azurite for local development                                   |
+| AZURITE_HOST              | http://fcp-fd-file-processor-azurite-development       | No                        | Azurite host name                                                          |
```

Comment:
Really like this table and have used this on personal GitHub repos. Think this is definitely something we should have across all our repos that we update continuously. Not sure if it's worth putting a ticket under `tech debt`/`continuous improvement` on the backlog to do the same for the Comms and Data services? Obviously for both services there are also a few sensitive values that we can just put down as `(no default value)` as you've done already, but referencing them the same way you have here will be needed for onboarding new devs/the distant future when SFD is handed over.

I remember discussing READMEs with @FayToward during a previous away day and what information they should contain so definitely something to just make standard across all our repos.

Reply:
Discussed with @sfitz42, will get something on the backlog.